### DISCUSSION TITLE
"Upload proof of your academic status" just reloads after sending

### DISCUSSION THREAD

**Discussion by KokosTech** [community/community#113670] [GitHub Education] [answered]
### Hi everyone,

Hello,

I am trying to reverify my school affiliation using the "Upload proof of your academic status" but it has been unsuccessful. After submitting and waiting for the status bar to fill up (percentages) to 100%, it just redirects me to the same page, without any feedback nor email.

**Comment by stefanbadea-sudo** [comment]
I have the same problem...

**Comment by SebastienXia** [comment]
The upload option doesn't work, you need to proof your academic status with your camera phone. It's what I do for this problem.

**Reply by KokosTech** [reply]
That's what I've tried doing, but it just loads to a 100 percent and then redirects me to the same page.

**Comment by ThatPaple** [comment]
I have a solution to this, use your school email (if it's a university email)!
If you have a proof of enrollment, you should be able to upload it, and if it does not work, try creating a [support ticket](https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket).

For me and a lot of my colleagues, the solution of "use university email" worked the most, you can still use the same account, if you add your university email as a secondary email. If using a university email is not a possibility, I'd recommend trying the proof of enrollment, and a ticket.

**Reply by KokosTech** [reply]
I have added my school email, but it has always asked me to give proof of enrolment, because it doesn't support my organisation. I have tried countless times to prove it using the form.

I'll try to create a support ticket, thanks

**Comment by KokosTech** [comment, marked as answer]
So, the solution is.... *drum rolls*

Magic, blackbox, or whatever you wanna call it. Technically there is no solution, you just have to wait a random amount of time and try filling the form again. It might work, it might not - it's up to your luck.

But I finally sent mine without a hitch.

(after "sending" (or at the very least - trying to do so) form 20 times) 

So if you haven't resolved yours (@stefanbadea-sudo), I would suggest trying it now.

**Reply by stefanbadea-sudo** [reply]
Thanks, for me it worked using the webcam on my laptop to submit the application 🫠
